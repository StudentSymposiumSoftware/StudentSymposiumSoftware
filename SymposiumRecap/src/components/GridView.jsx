import "./GridView.css"

function GridView() {
    return (
        <div id="grid-container">
            Welcome to the grid view page
        </div>
    )
}

export default GridView
  