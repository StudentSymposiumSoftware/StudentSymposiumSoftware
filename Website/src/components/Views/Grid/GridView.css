.grid-container{
    display: grid;
    grid-template-columns: auto auto auto auto auto;
    grid-gap: 12px;
    padding: 30px;
    width:auto;
}
.item {
    min-height: 200px;
    background-color: #aab3de;
    color: black;
    text-align: center;
    width:auto;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    padding: 5px;
    border-radius: 10px;
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
}
.item:hover{
    transform: scale(1.05);
    transition: transform 0.2s ease-in-out;
    cursor: pointer;
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
}

.author-link {
    text-decoration: underline;
    color: rgb(31, 31, 101);
    cursor: pointer;
    transition: color 0.2s ease-in-out;
}

.author-link:hover {
    color: darkblue;
}

.author-link:active {
    color: red;
}