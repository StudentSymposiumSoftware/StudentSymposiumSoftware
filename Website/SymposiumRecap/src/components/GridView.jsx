import "./GridView.css"

function GridView(props) {
    
    console.log(props.data) // <-- Here's the data for your ticket

    return (
        <div id="grid-container">
            Welcome to the grid view page
        </div>
    )
}

export default GridView
  