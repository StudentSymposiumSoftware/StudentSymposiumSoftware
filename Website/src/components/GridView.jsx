import "./GridView.css"
import { useNavigate } from "react-router-dom"

function GridView(props) {
    const navigate = useNavigate();
    
    return (
        <div className="grid-container">
            {props.data.map((item, index) => (
                <div key={index} className="item" >
  
                    <h4>{item.title}</h4>
                    <h4 onClick={() => navigate(`/author/${item.author}`)} className="author-link">By {item.author}</h4>
                    <p>{[item.major, 'at', item.school].join(' ')}</p>
                    <p>{['Abstract number', item.abstractNumber].join(' ')}</p>
                </div>
      ))}
        </div>
    );


}

export default GridView
