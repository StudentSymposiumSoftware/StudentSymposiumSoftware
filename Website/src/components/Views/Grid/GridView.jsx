import "./GridView.css"
import { toTitleCase, findSchoolLogo, abstractDataSearch } from "../../shared"
import PersonIcon from '@mui/icons-material/PersonRounded';
import { useNavigate } from "react-router-dom";

function GridView({data, searchQuery, setSearchText}) {
    data = data.filter(item => {
        if (searchQuery === "") return true;
        return abstractDataSearch(item, searchQuery);
    });
    return (
        <div className='grid-container'>
            {data == 0 && <span className="loader flex place-self-center col-span-5"/>}
            {data.map((item, index) => (<AbstractGrid key={index} item={item} index={index} setSearchText={(e) => setSearchText(e)}/>))}
        </div>
    );
}

function AbstractGrid({item, index, setSearchText}) {
    const navigate = useNavigate();
    return (
        <a key={index} className="item" href={`#/abstract/${item.abstractNumber}`}>
            <img src={findSchoolLogo(item.school)} alt="school logo" className="mt-5 p-[2px]"/>
            <h4>{`${item.abstractNumber}: ${toTitleCase(item.title)}`}</h4>
            <span><PersonIcon className="mr-[2px]"/><h4 onClick={() => setSearchText(item.author)} className="author-link inline-block leading-[24px]"> {toTitleCase(item.author)}</h4></span>
        </a>
    )
}

export default GridView