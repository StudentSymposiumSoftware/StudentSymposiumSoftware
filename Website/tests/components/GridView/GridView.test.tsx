import React from 'react';
import { render } from "@testing-library/react";
import { expect, it } from "vitest";

import GridView from '../../../src/components/GridView';
import mockData from '../../../src/mockData'

it('GridView renders correctly', () => {
    const tree = render(<GridView data={mockData}/>)
    expect(tree).toMatchSnapshot();
});