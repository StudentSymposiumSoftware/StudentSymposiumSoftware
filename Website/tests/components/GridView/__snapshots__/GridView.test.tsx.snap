// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GridView renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="grid-container"
      >
        <div
          class="item"
        >
          <h4>
            Machine Learning Approaches to Financial Fraud Detection
          </h4>
          <h4
            class="author-link"
          >
            By 
            Elijah Smith
          </h4>
          <p>
            Physics at UMaine Machias
          </p>
          <p>
            Abstract number 38893888
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Psychology of Decision-Making in Uncertain Environments
          </h4>
          <h4
            class="author-link"
          >
            By 
            Sophia Campbell
          </h4>
          <p>
            Political Science at UMaine Machias
          </p>
          <p>
            Abstract number 75957852
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Sociology of Online Communities in the Digital Age
          </h4>
          <h4
            class="author-link"
          >
            By 
            Lucy Martinez
          </h4>
          <p>
            Business Administration at UMaine Machias
          </p>
          <p>
            Abstract number 48203955
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Zoe Jackson
          </h4>
          <p>
            Psychology at UMaine Machias
          </p>
          <p>
            Abstract number 64568884
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Blockchain Technology in Supply Chain Transparency
          </h4>
          <h4
            class="author-link"
          >
            By 
            Noah King
          </h4>
          <p>
            Political Science at UMaine Machias
          </p>
          <p>
            Abstract number 40679302
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Joseph Baker
          </h4>
          <p>
            Mathematics at UMaine Orono
          </p>
          <p>
            Abstract number 37625940
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Climate Change and Its Effects on Global Food Security
          </h4>
          <h4
            class="author-link"
          >
            By 
            Ava Rodriguez
          </h4>
          <p>
            Physics at UMaine Machias
          </p>
          <p>
            Abstract number 35716422
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Psychology of Decision-Making in Uncertain Environments
          </h4>
          <h4
            class="author-link"
          >
            By 
            Henry Adams
          </h4>
          <p>
            Psychology at UMaine Machias
          </p>
          <p>
            Abstract number 82413071
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Machine Learning Approaches to Financial Fraud Detection
          </h4>
          <h4
            class="author-link"
          >
            By 
            Mason Hernandez
          </h4>
          <p>
            Business Administration at UMaine Orono
          </p>
          <p>
            Abstract number 27882383
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Machine Learning Approaches to Financial Fraud Detection
          </h4>
          <h4
            class="author-link"
          >
            By 
            Emma Taylor
          </h4>
          <p>
            Biology at UMaine Orono
          </p>
          <p>
            Abstract number 92969314
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Role of Citizen Science in Biodiversity Conservation
          </h4>
          <h4
            class="author-link"
          >
            By 
            Aurora Martinez
          </h4>
          <p>
            Computer Science at UMaine Machias
          </p>
          <p>
            Abstract number 29718536
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Role of Citizen Science in Biodiversity Conservation
          </h4>
          <h4
            class="author-link"
          >
            By 
            Lucas Robinson
          </h4>
          <p>
            Psychology at UMaine Orono
          </p>
          <p>
            Abstract number 1457768
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Blockchain Technology in Supply Chain Transparency
          </h4>
          <h4
            class="author-link"
          >
            By 
            Chloe Adams
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 14148682
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Exploring the Feasibility of Space-Based Solar Power
          </h4>
          <h4
            class="author-link"
          >
            By 
            Layla Young
          </h4>
          <p>
            Biology at UMaine Orono
          </p>
          <p>
            Abstract number 60454722
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Social Media Influence on Adolescent Mental Health
          </h4>
          <h4
            class="author-link"
          >
            By 
            Alexander Hall
          </h4>
          <p>
            Business Administration at UMaine Machias
          </p>
          <p>
            Abstract number 41498009
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Analyzing the Impact of Microplastics on Marine Ecosystems
          </h4>
          <h4
            class="author-link"
          >
            By 
            Zoe Young
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 23333306
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Cultural Impacts on Language Evolution
          </h4>
          <h4
            class="author-link"
          >
            By 
            Mason Clark
          </h4>
          <p>
            Nursing at UMaine Machias
          </p>
          <p>
            Abstract number 97656877
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Social Media Influence on Adolescent Mental Health
          </h4>
          <h4
            class="author-link"
          >
            By 
            Henry Hall
          </h4>
          <p>
            Mechanical Engineering at UMaine Machias
          </p>
          <p>
            Abstract number 91417686
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Samuel Scott
          </h4>
          <p>
            Mathematics at UMaine Machias
          </p>
          <p>
            Abstract number 4887031
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Grace Nelson
          </h4>
          <p>
            Nursing at UMaine Machias
          </p>
          <p>
            Abstract number 36475226
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Machine Learning Approaches to Financial Fraud Detection
          </h4>
          <h4
            class="author-link"
          >
            By 
            Scarlett Hernandez
          </h4>
          <p>
            Mechanical Engineering at UMaine Machias
          </p>
          <p>
            Abstract number 29014475
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Future of CRISPR in Gene Therapy
          </h4>
          <h4
            class="author-link"
          >
            By 
            Maya Hernandez
          </h4>
          <p>
            Physics at UMaine Machias
          </p>
          <p>
            Abstract number 44667664
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Analyzing the Impact of Microplastics on Marine Ecosystems
          </h4>
          <h4
            class="author-link"
          >
            By 
            Elijah Rivera
          </h4>
          <p>
            Computer Science at UMaine Orono
          </p>
          <p>
            Abstract number 21709418
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Social Media Influence on Adolescent Mental Health
          </h4>
          <h4
            class="author-link"
          >
            By 
            Liam Taylor
          </h4>
          <p>
            Political Science at UMaine Machias
          </p>
          <p>
            Abstract number 45822327
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            The Impact of Remote Work on Organizational Productivity
          </h4>
          <h4
            class="author-link"
          >
            By 
            Mia Robinson
          </h4>
          <p>
            Mathematics at UMaine Orono
          </p>
          <p>
            Abstract number 75710656
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Deep Learning for Medical Image Analysis
          </h4>
          <h4
            class="author-link"
          >
            By 
            Evelyn Rodriguez
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 39877467
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Cultural Impacts on Language Evolution
          </h4>
          <h4
            class="author-link"
          >
            By 
            Zoe Brown
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 74596190
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Sophie Thomas
          </h4>
          <p>
            Mechanical Engineering at UMaine Machias
          </p>
          <p>
            Abstract number 30556122
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Advancements in Human-Robot Interaction for Assistive Technologies
          </h4>
          <h4
            class="author-link"
          >
            By 
            Maya Mitchell
          </h4>
          <p>
            Nursing at UMaine Orono
          </p>
          <p>
            Abstract number 75180428
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Analyzing the Impact of Microplastics on Marine Ecosystems
          </h4>
          <h4
            class="author-link"
          >
            By 
            Jack Anderson
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 82398661
          </p>
        </div>
        <div
          class="item"
        >
          <h4>
            Analyzing the Impact of Microplastics on Marine Ecosystems
          </h4>
          <h4
            class="author-link"
          >
            By 
            Harper Rodriguez
          </h4>
          <p>
            Physics at UMaine Orono
          </p>
          <p>
            Abstract number 82398661
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="grid-container"
    >
      <div
        class="item"
      >
        <h4>
          Machine Learning Approaches to Financial Fraud Detection
        </h4>
        <h4
          class="author-link"
        >
          By 
          Elijah Smith
        </h4>
        <p>
          Physics at UMaine Machias
        </p>
        <p>
          Abstract number 38893888
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Psychology of Decision-Making in Uncertain Environments
        </h4>
        <h4
          class="author-link"
        >
          By 
          Sophia Campbell
        </h4>
        <p>
          Political Science at UMaine Machias
        </p>
        <p>
          Abstract number 75957852
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Sociology of Online Communities in the Digital Age
        </h4>
        <h4
          class="author-link"
        >
          By 
          Lucy Martinez
        </h4>
        <p>
          Business Administration at UMaine Machias
        </p>
        <p>
          Abstract number 48203955
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Zoe Jackson
        </h4>
        <p>
          Psychology at UMaine Machias
        </p>
        <p>
          Abstract number 64568884
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Blockchain Technology in Supply Chain Transparency
        </h4>
        <h4
          class="author-link"
        >
          By 
          Noah King
        </h4>
        <p>
          Political Science at UMaine Machias
        </p>
        <p>
          Abstract number 40679302
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Joseph Baker
        </h4>
        <p>
          Mathematics at UMaine Orono
        </p>
        <p>
          Abstract number 37625940
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Climate Change and Its Effects on Global Food Security
        </h4>
        <h4
          class="author-link"
        >
          By 
          Ava Rodriguez
        </h4>
        <p>
          Physics at UMaine Machias
        </p>
        <p>
          Abstract number 35716422
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Psychology of Decision-Making in Uncertain Environments
        </h4>
        <h4
          class="author-link"
        >
          By 
          Henry Adams
        </h4>
        <p>
          Psychology at UMaine Machias
        </p>
        <p>
          Abstract number 82413071
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Machine Learning Approaches to Financial Fraud Detection
        </h4>
        <h4
          class="author-link"
        >
          By 
          Mason Hernandez
        </h4>
        <p>
          Business Administration at UMaine Orono
        </p>
        <p>
          Abstract number 27882383
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Machine Learning Approaches to Financial Fraud Detection
        </h4>
        <h4
          class="author-link"
        >
          By 
          Emma Taylor
        </h4>
        <p>
          Biology at UMaine Orono
        </p>
        <p>
          Abstract number 92969314
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Role of Citizen Science in Biodiversity Conservation
        </h4>
        <h4
          class="author-link"
        >
          By 
          Aurora Martinez
        </h4>
        <p>
          Computer Science at UMaine Machias
        </p>
        <p>
          Abstract number 29718536
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Role of Citizen Science in Biodiversity Conservation
        </h4>
        <h4
          class="author-link"
        >
          By 
          Lucas Robinson
        </h4>
        <p>
          Psychology at UMaine Orono
        </p>
        <p>
          Abstract number 1457768
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Blockchain Technology in Supply Chain Transparency
        </h4>
        <h4
          class="author-link"
        >
          By 
          Chloe Adams
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 14148682
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Exploring the Feasibility of Space-Based Solar Power
        </h4>
        <h4
          class="author-link"
        >
          By 
          Layla Young
        </h4>
        <p>
          Biology at UMaine Orono
        </p>
        <p>
          Abstract number 60454722
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Social Media Influence on Adolescent Mental Health
        </h4>
        <h4
          class="author-link"
        >
          By 
          Alexander Hall
        </h4>
        <p>
          Business Administration at UMaine Machias
        </p>
        <p>
          Abstract number 41498009
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Analyzing the Impact of Microplastics on Marine Ecosystems
        </h4>
        <h4
          class="author-link"
        >
          By 
          Zoe Young
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 23333306
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Cultural Impacts on Language Evolution
        </h4>
        <h4
          class="author-link"
        >
          By 
          Mason Clark
        </h4>
        <p>
          Nursing at UMaine Machias
        </p>
        <p>
          Abstract number 97656877
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Social Media Influence on Adolescent Mental Health
        </h4>
        <h4
          class="author-link"
        >
          By 
          Henry Hall
        </h4>
        <p>
          Mechanical Engineering at UMaine Machias
        </p>
        <p>
          Abstract number 91417686
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Samuel Scott
        </h4>
        <p>
          Mathematics at UMaine Machias
        </p>
        <p>
          Abstract number 4887031
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Grace Nelson
        </h4>
        <p>
          Nursing at UMaine Machias
        </p>
        <p>
          Abstract number 36475226
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Machine Learning Approaches to Financial Fraud Detection
        </h4>
        <h4
          class="author-link"
        >
          By 
          Scarlett Hernandez
        </h4>
        <p>
          Mechanical Engineering at UMaine Machias
        </p>
        <p>
          Abstract number 29014475
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Future of CRISPR in Gene Therapy
        </h4>
        <h4
          class="author-link"
        >
          By 
          Maya Hernandez
        </h4>
        <p>
          Physics at UMaine Machias
        </p>
        <p>
          Abstract number 44667664
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Analyzing the Impact of Microplastics on Marine Ecosystems
        </h4>
        <h4
          class="author-link"
        >
          By 
          Elijah Rivera
        </h4>
        <p>
          Computer Science at UMaine Orono
        </p>
        <p>
          Abstract number 21709418
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Social Media Influence on Adolescent Mental Health
        </h4>
        <h4
          class="author-link"
        >
          By 
          Liam Taylor
        </h4>
        <p>
          Political Science at UMaine Machias
        </p>
        <p>
          Abstract number 45822327
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          The Impact of Remote Work on Organizational Productivity
        </h4>
        <h4
          class="author-link"
        >
          By 
          Mia Robinson
        </h4>
        <p>
          Mathematics at UMaine Orono
        </p>
        <p>
          Abstract number 75710656
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Deep Learning for Medical Image Analysis
        </h4>
        <h4
          class="author-link"
        >
          By 
          Evelyn Rodriguez
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 39877467
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Cultural Impacts on Language Evolution
        </h4>
        <h4
          class="author-link"
        >
          By 
          Zoe Brown
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 74596190
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Sophie Thomas
        </h4>
        <p>
          Mechanical Engineering at UMaine Machias
        </p>
        <p>
          Abstract number 30556122
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Advancements in Human-Robot Interaction for Assistive Technologies
        </h4>
        <h4
          class="author-link"
        >
          By 
          Maya Mitchell
        </h4>
        <p>
          Nursing at UMaine Orono
        </p>
        <p>
          Abstract number 75180428
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Analyzing the Impact of Microplastics on Marine Ecosystems
        </h4>
        <h4
          class="author-link"
        >
          By 
          Jack Anderson
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 82398661
        </p>
      </div>
      <div
        class="item"
      >
        <h4>
          Analyzing the Impact of Microplastics on Marine Ecosystems
        </h4>
        <h4
          class="author-link"
        >
          By 
          Harper Rodriguez
        </h4>
        <p>
          Physics at UMaine Orono
        </p>
        <p>
          Abstract number 82398661
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
